### PR TITLE
refactor: rework webviews

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves []

### Description

`ArtsyWebView` -> `ArtsyWebViewPage` (screen with a back button and a webview)
`ArtsyReactWebViewPage` -> `ArtsyWebViewPage` (exact same as above)
`ArtsyReactWebView` -> `ArtsyWebView` (the actual webview)

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/AppRegistry.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { GraphQLTaggedNode } from "relay-runtime"
 import { SafeAreaInsets, useScreenDimensions } from "shared/hooks"
 import { ArtsyKeyboardAvoidingViewContext } from "shared/utils"
 import { AppProviders } from "./AppProviders"
-import { ArtsyReactWebViewPage } from "./Components/ArtsyReactWebView"
+import { ArtsyWebViewPage } from "./Components/ArtsyWebView"
```
Comment (brainbicycle) [inline src/app/AppRegistry.tsx:10]: nice removing React makes sense ✨

**Review by brainbicycle — APPROVED** [review]
I made some modifications to the other pr this is on top of so this may need a rebase sorry! but looks good to me, thanks for cleaning this up!

**Comment by pvinis** [comment]
i can rebase tomorrow

**Comment by pvinis** [comment]
rebased 👍

**Pull request merged**